images/resize.mjs

```diff
@@ -0,0 +1,79 @@
+import sharp from 'sharp'
+import fs from 'fs'
+import path from 'path'
+import prettyBytes from 'pretty-bytes'
+
+const SUPPORTED_IMAGE_EXTENSIONS = ['jpeg', 'jpg', 'png']
+
+const imageFiles = fs
+  .readdirSync('.')
+  .filter(
+    file =>
+      !file.includes('-thumb') &&
+      SUPPORTED_IMAGE_EXTENSIONS.some(ext => file.includes(ext))
+  )
```

Comment:
Could we also add a filter to remove images that already have a thumbnail?

Reply:
Ahh... That makes sense that we could update the actual images.. In that case, I think retaining it as it is right now, would be lesser hassle..